src/yunohost/app.py

```diff
@@ -239,6 +239,8 @@ def _sanitized_absolute_url(perm_url):
                 perm_domain, perm_path = perm_url.split("/", 1)
                 perm_path = "/" + perm_path.rstrip("/")
 
+            perm_path = perm_path if perm_path != "" else "/"
```

Comment:
```suggestion
            perm_path = perm_path if perm_path.strip() != "" else "/"
```

Not sure it's needed but if that can prevent a bug...